portal/src/views/projects/ProjectForm.vue

```diff
@@ -140,6 +140,8 @@ import { helpers, required, email, minValue, maxLength, minLength } from "vuelid
 import FKApi from "@/api/api";
 import * as ActionTypes from "@/store/actions";
 
+import PlaceholderImage from "../../assets/image-placeholder.svg";
```

Comment:
Hey can you do my a favor and run 'make' before you submit PRs and make sure that it passes? I've had a few PRs fail the CI lately.

Reply:
Sure